ci-operator/step-registry/optional-operators/cvp-common/scorecard/optional-operators-cvp-common-scorecard-commands.sh

```diff
@@ -4,28 +4,32 @@ set -o nounset
 set -o errexit
 set -o pipefail
 
+# This value serves as a default when the parameters are not set, which should
+# only happen in rehearsals. Production jobs should always set the OO_* variable.
+REHEARSAL_BUNDLE="brew.registry.redhat.io/rh-osbs/rhacm2-acm-operator-bundle:v2.1.5-6"
```

Comment:
Why not set these as default values on the param field?

Reply:
Because they are not multi-stage parameters (these steps predate step parameters). They currently cannot be b/c cvp-trigger needs to set them dynamically, not statically in ci-op config. We need to make ci-ope accept params set externally for that, that's a different work item.